src/applications/check-in/day-of/pages/Confirmation/index.jsx

```diff
@@ -1,14 +1,33 @@
-import React, { useCallback, useMemo } from 'react';
+import React, { useCallback, useEffect, useMemo } from 'react';
+import PropTypes from 'prop-types';
 import { useDispatch, useSelector } from 'react-redux';
 
+import { api } from '../../../api';
 import MultipleAppointment from './MultipleAppointments';
-
 import { triggerRefresh } from '../../../actions/day-of';
-
 import { makeSelectConfirmationData } from '../../../selectors';
+import { makeSelectFeatureToggles } from '../../../utils/selectors/feature-toggles';
+
+import { useSessionStorage } from '../../../hooks/useSessionStorage';
+import { useDemographicsFlags } from '../../../hooks/useDemographicsFlags';
+import { useFormRouting } from '../../../hooks/useFormRouting';
 
-const Confirmation = () => {
+const Confirmation = props => {
```

Comment:
It might be better to destructure `props` to `({ router }}` here so you can get rid of line 16 below